errata_tool/connector.py

```diff
@@ -310,3 +310,37 @@ def get_paginated_data(self, api_url):
             if page_number >= PAGE_LIMIT:
                 raise RuntimeError('hit pagination timeout: %d' % page_number)
         return data
+
+    def filter(self, endpoint, filter_arg, **kwargs):
```

Comment:
How about naming this "get_filtered()" instead? "get" tells us that it's a "GET" request

Reply:
makes sense to me and good refinement!